test/interpreter.ts

```diff
@@ -114,3 +114,40 @@ describe('error location', () => {
 		`)).resolves.toEqual({ line: 2, column: 6});
 	});
 });
+
+describe('callstack', () => {
+	const exeAndGetErrMessage = (src: string): Promise<string> => new Promise((ok, ng) => {
+		const aiscript = new Interpreter({
+			emitError: FN_NATIVE((_args, _opts) => {
+				throw Error('emitError');
+			}),
+		}, {
+			err(e) { ok(e.message) },
+		});
+		aiscript.exec(Parser.parse(src)).then(() => ng('error has not occurred.'));
+	});
+
+	test('error in function', async () => {
+		const result = await exeAndGetErrMessage(`
+			@function1() { emitError() }
+			@function2() { function1() }
+			function2()
+		`);
+		expect(result).toMatch(/at function1.+at function2/s);
```

Comment:
判定をもう少し正確にする方がテストとして安全だと思います。
完全一致の場合：
```suggestion
		expect(result).toEqual("emitError\n  at function1 (Line 2, Column 28)\n  at function2 (Line 3, Column 28)\n  at <root> (Line 4, Column 13)");
```

Reply:
修正しました